frontend/src/lib/entities/questions/Number.ts

```diff
@@ -24,6 +24,6 @@ export class NumberQuestionAnswered extends NumberQuestion {
 	}
 
 	getAnswer(): string {
-		return this.answer ? this.answer.toString() : '';
+		return !isNaN(this.answer) ? this.answer.toString() : '';
```

Comment:
please let's also do that for all other non-number types just to be sure and to keep it consistent